### PR TITLE
Update azure-virtual-desktop-multi-session.md

### PR DESCRIPTION
**Pull request by SamzerOnGit**
Comment on Co-management, like here https://docs.microsoft.com/en-us/mem/intune/protect/certificates-profile-scep#assign-the-certificate-profile

### CODE REVIEW

**Comment by PRMerger10** [comment]
@SamzerOnGit : Thanks for your contribution! The author(s) have been notified to review your proposed change. @smbhardwaj

File: memdocs/intune/fundamentals/azure-virtual-desktop-multi-session.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ The existing device configuration profile templates aren't supported for Windows
 
 Microsoft Intune won't deliver unsupported templates to multi-session devices, and those policies appear as *Not applicable* in reports.
 
+> [!NOTE] If you use co-management for Intune and Configuration Manager, in Configuration Manager [set the workload slider](/configmgr/comanage/how-to-switch-workloads) for Resource Access Policies to **Intune** or **Pilot Intune**. This setting allows Windows 10/11 clients to start the process of requesting the certificate.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/azure-virtual-desktop-multi-session.md:86]:
```suggestion
> [!NOTE]
> If you use comanagement for Intune and Configuration Manager, in Configuration Manager, [set the workload slider](/configmgr/comanage/how-to-switch-workloads) for Resource Access Policies to **Intune** or **Pilot Intune**. This setting allows Windows 10 and Windows 11 clients to start the process of requesting the certificate.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @SamzerOnGit, kindly commit the suggested changes. Many thanks.

**Comment by Smritib17** [comment]
#sign-off

**Pull request merged**